teachertool/src/App.tsx

```diff
@@ -54,17 +51,12 @@ function App() {
             <div className="ui large main loader msft"></div>
         </div>
     ) : (
-        <div className="app-container">
+        <>
```

Comment:
What's the benefit of making this an empty tag rather than a div?

Reply:
The empty tag is shorthand for React Fragment. I moved the functionality this div was providing into MainPanel. But we still need a container here as a workaround for React's limitation of being unable to render a list of nodes at the top level of a component. React Fragments well-suited for this, because they make clear that the purpose of the node isn't to express UI, but to act solely as a container.

Reply:
Oh, interesting, thanks for the information!